src/main/java/org/mule/service/http/impl/service/server/HttpServerManager.java

```diff
@@ -69,6 +87,26 @@ HttpServer createSslServerFor(TlsContextFactory tlsContextFactory, Supplier<Sche
                                 ServerIdentifier identifier, Supplier<Long> shutdownTimeout)
       throws ServerCreationException;
 
+  /**
+   *
+   * @param tlsContextFactory
+   * @param schedulerSupplier        work manager source to use for retrieving an {@link Executor} for processing this server
+   *                                 requests
+   * @param serverAddress            address of the server
+   * @param usePersistentConnections if true, the connections will be kept open for subsequent requests
+   * @param connectionIdleTimeout    the amount of milliseconds to keep open an idle connection
+   * @param identifier               the id of the server
+   * @param shutdownTimeout          time to wait for persistent connections to be closed when server is stopped.
+   * @param readTimeout              time to wait while reading input in milliseconds
+   * @return the create Server handler
+   * @throws ServerCreationException if it was not possible to create the Server. Most likely because the host and port is already
+   *                                 in use.
+   */
+  HttpServer createSslServerFor(TlsContextFactory tlsContextFactory, Supplier<Scheduler> schedulerSupplier,
```

Comment:
Deprecate the equivalent method without the readTimeout (don't delete them, just deprecate)